_posts/2020-05-27-ols1-wrapup.md

```diff
@@ -1,6 +1,9 @@
 ---
 layout: post
 title: End of OLS-1 and OLS-2 Announcement post
+categories: News
+authors: 
+=======
```

Comment:
This line is creating a local exception on my side, @oluwatobiBolu could you check it?